### PR TITLE
COMMERCE-11970

### PR DESCRIPTION
**Pull request by 22LucaGiannotta22**
Resent from #4030 

https://liferay.atlassian.net/browse/LPP-49903

### CODE REVIEW

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase [hunk 1]
```diff
@@ -246,6 +246,62 @@ definition {
 			locator1 = "CommerceAccelerators#PRODUCT_CARD_NAME");
 	}
 
+	@description = "This is a test case for COMMERCE-11970. As an admin, I want that the term count of price range facet fields doesn't change to 0 when I check a price range facet field"
+	@priority = 4
+	test AssertTermCountOfPriceRangeFacetFieldsDoesNotChange {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase:251]: CanViewTermCountOfPriceRangeFacetPersist

**Review by baharturk — CHANGES_REQUESTED** [review]
Please take a look at my comments. Thanks!

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Review by baharturk — APPROVED** [review]

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
relevant failures are not related to the PR. This can be force forwarded

**Pull request closed without merging**